src/KOKKOS/out.txt

```diff
@@ -0,0 +1,44 @@
+pair_buck_coul_cut_kokkos.cpp
```

Comment:
probably these two files shouldn't be here

Reply:
Yes good catch, I removed them.